doc/mf6io/mf6ivar/dfn/gwf-ghba.dfn

```diff
@@ -0,0 +1,159 @@
+reader urword
+optional true
+longname print calculated flows to listing file
+description REPLACE print_flows {'{#1}': 'general-head boundary'}
+mf6internal iprflow
+
+block options
+name save_flows
+type keyword
+reader urword
+optional true
+longname save CHD flows to budget file
+description REPLACE save_flows {'{#1}': 'general-head boundary'}
+mf6internal ipakcb
+
+block options
+name obs_filerecord
+type record obs6 filein obs6_filename
+shape
+reader urword
+tagged true
+optional true
+longname
+description
+
+block options
+name obs6
+type keyword
+shape
+in_record true
+reader urword
+tagged true
+optional false
+longname obs keyword
+description keyword to specify that record corresponds to an observations file.
+
+block options
+name filein
+type keyword
+shape
+in_record true
+reader urword
+tagged true
+optional false
+longname file keyword
+description keyword to specify that an input filename is expected next.
+
+block options
+name obs6_filename
+type string
+preserve_case true
+in_record true
+tagged false
+reader urword
+optional false
+longname obs6 input filename
+description REPLACE obs6_filename {'{#1}': 'General-Head Boundary'}
+
+block options
+name mover
+type keyword
+tagged true
+reader urword
+optional true
+longname
+description REPLACE mover {'{#1}': 'General-Head Boundary'}
+
+block options
+name export_array_netcdf
+type keyword
+reader urword
+optional true
+mf6internal export_nc
+longname export array variables to netcdf output files.
+description keyword that specifies input griddata arrays should be written to the model output netcdf file.
+extended true
+
+# --------------------- gwf ghba period ---------------------
+
+block period
+name iper
+type integer
+block_variable True
+in_record true
+tagged false
+shape
+valid
+reader urword
+optional false
+longname stress period number
+description REPLACE iper {}
+
+block period
+name bhead
+type double precision
+shape (nodes)
+reader readarray
+layered true
+netcdf true
+longname boundary head
+description is the boundary head.
+default_value 3.e30
+
+block period
+name cond
+type double precision
+shape (nodes)
+reader readarray
+layered true
+netcdf true
+longname boundary conductance
+description is the hydraulic conductance of the interface between the aquifer cell and the boundary.
+default_value 3.e30
+
+block period
+name aux
+type double precision
+shape (nodes)
+reader readarray
+layered true
+netcdf true
+optional true
+longname general-head boundary auxiliary variable iaux
+description is an array of values for auxiliary variable aux(iaux), where iaux is a value from 1 to naux, and aux(iaux) must be listed as part of the auxiliary variables.  A separate array can be specified for each auxiliary variable.  If an array is not specified for an auxiliary variable, then a value of zero is assigned.  If the value specified here for the auxiliary variable is the same as auxmultname, then the boundary head array will be multiplied by this array.
```

Comment:
I'm not necessarily suggesting we do this, but would a DNODATA value be better for an aux variable that is not assigned?